.circleci/main.yml

```diff
@@ -956,25 +947,22 @@ jobs:
       - run:
           name: Publish
           command: |
-            python -m venv venv
             . venv/bin/activate
             twine upload --repository testpypi dist/*
 
   jupyter-extension-publish:
     executor: python
+    resource_class: large
     working_directory: ~/project/jupyter-extension
     steps:
       - checkout:
           path: ~/project
       - attach_workspace:
           at: .
+      - install-bazel
       - run:
           name: "Install Python dependencies"
-          command: |
-            python -m venv venv
-            . venv/bin/activate
-            python -m pip install --upgrade pip
-            python -m pip install -r requirements-dev.txt
+          command: bazel run :venv
```

Comment:
This can be `bazel --config=remotecache --remote_header=x-buildbuddy-api-key=${BUILDBUDDY_API_KEY}` so we can grab as much of the action cache and object cache as possible... i.e. when venv deps don't change, this just downloads from the Internet instead of running anything.

Reply:
I assume these flags are before the `run :venv` command, right?

Reply:
yup yup